screw_control:
  - name: m5_tool              # Name of screw fastening tool
    digital_in_port: 0         # The pin to which the pressure sensor is connected
    digital_out_port_vac: 0    # The pin to which the 'vac' of vacuum pump unit is connected
    digital_out_port_blow: 1   # The pin to which the 'blow' of vacuum pump unit is connected
  - name: m4_tool
    digital_in_port: 1
    digital_out_port_vac: 1
    digital_out_port_blow: 2
  - name: m3_tool
    digital_in_port: 2
    digital_out_port_vac: 2
    digital_out_port_blow: 3

